frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -37,11 +37,11 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
 
 # Base actions
 
```

Comment:
Hmm I'd also apply the changes to the:
1. docs (only to text, IMO no need to regenerate the screencasts)
1. action classes
1. UI tests
(at least I can think of those 3 scopes now)